xwayland/xwm.h

```diff
@@ -0,0 +1,84 @@
+#ifndef XWAYLAND_INTERNALS_H
+#define XWAYLAND_INTERNALS_H
+#include <wayland-server-core.h>
+#include <wlr/xwayland.h>
+
+/* wlc's atom list:
```

Comment:
Weston does have the advantage that there is no enum in the .h, but there is that huge temporary struct within a function that I don't like at all.
There also is an hard-coded list of atoms in the header rightin the weston_wm struct when we just have an ATOM_LAST-sized array there -- I guess it's down to taste, but I prefer our approach that just defines the enum and strings, so we have them all at the same place, and it's easy to loop over.
Wish I could define the two at the same time to ensure no mismatch problem though, will check if there's any trick to that I think I might have seen it somewhere...